NotifyMe/views/views.py

```diff
@@ -0,0 +1,324 @@
+import logging
+from django.core.exceptions import PermissionDenied
+from django.db import IntegrityError
+from django.core.exceptions import ValidationError
+from rest_framework.response import Response
+from ..serializers import UserSerializer, SubscriptionSerializer, SubscriptionPlanSerializer
+from NotifyMe.models.user import User
+from NotifyMe.models.subscription import Subscription
+from NotifyMe.models.subscriptionPlan import SubscriptionPlan
+from rest_framework import status 
+from rest_framework.views import APIView
+from NotifyMe.services.service import UserService, SubscriptionService, SubscriptionPlanService
+
+
+
+logger = logging.getLogger(__name__)
+
+
+# ---------USER-API---------------- 
+# DONOT CALL MODEL INSIDE VIEW---> ONLY CALL SERVICE AND SERIALIZER
+
+def get_response_data(success, message=None, data=None):
```

Comment:
why this method is outside of the class , what is the purpose of this im not observing the purpose from its name in case of error what you will do

Reply:
I have used this to return Resposne message of same structure . This method is used in every Response in view. It returns an uniform Response message

Reply:
No method should not be outside of a class  it is not handling in error